src/SayMore/UI/ProjectWindow/ProjectWindow.cs

```diff
@@ -45,6 +45,8 @@ public partial class ProjectWindow : Form
 		private MPlayerDebuggingOutputWindow _outputDebuggingWindow;
 		private string _titleFmt;
 
+		private static string[] _keyDownTypes = {"TextBox", "ComboBox", "DatePicker"};
```

Comment:
the name _keyDownTypes isn't very clear.  Perhaps you can come up with a name that better conveys the intended meaning. A better approach might be to add a small static class with an extension method:
`static bool ShouldTreatEnterAsTab(this Control ctrl) // alternative name: ShouldEnterAdvanceToNextControl`
Your implementation could then just ask is ctrl TextBox, ComboBox, etc.. This would be more efficient than doing a string comparison on type name and would nicely extend to derived types. The implementation could also nicely deal with the special case of a Multiline text box (without having to check its type twice).